keras_cv/models/backbones/backbone.py

```diff
@@ -53,19 +53,25 @@ def presets(cls):
 
     @classproperty
     def presets_with_weights(cls):
-        """Dictionary of preset names and configurations that include
-        weights."""
+        """Dictionary of preset names and configs that include weights."""
         return {}
 
+    @classproperty
+    def presets_without_weights(cls):
```

Comment:
Heck yeah -- I dig this 😎 
Thanks!

Reply:
Actually I forgot to put this on the Task as well!!!